### PR TITLE
Doc ja

### PR DESCRIPTION
**Pull request by boiledorange73**
Japanese translations, excepting Tiger Geocoder, Address Standardizer and Release Notes.

### CODE REVIEW

**Comment by strk** [comment]
Is this coming from https://www.transifex.com/postgis/postgis/language/ja/ ?

**Comment by boiledorange73** [comment]
Almost Yes. Almost of translations are brought from transefix directly, and some of translations on github were modified to fit github document.

**Comment by strk** [comment]
I asked because I tried to make transifex downloads automated,
see doc/Makefile, the tx-pull and tx-push rules

Do they work for you ?

It's been some time so I'm not sure anymore how they work,
but the idea was to be able to fetch translations from there
and push any change (often needed once the translations are
checked to be valid, with "make check" under doc/).

What we really need badly is a section about transifex management
in the README over there

**Comment by boiledorange73** [comment]
I did not properly read README ... Sorry ...

I tried to "tx-pull" "gmake check" and "gmake html". They worked very fine !

Perhaps, it is useless to modify some of translations from transefix to fit the document on github ... ?

**Comment by strk** [comment]
Updating the transifex copy from the copy in the official repository (of which github code is a mirror) is useful when the official repository text is more correct than the one on transifex, so to reduce the risk of contributors translating again an already-translated text.

I cannot judge myself which side of the Japanese translation is more recent.

File: doc/Makefile.in [hunk 1]
```diff
@@ -182,8 +182,10 @@ update-po: $(XML_INPUTS_POT)
   done
 
 pull-tx:
-	tx pull --all
-	$(MAKE) update-po
+	@for lang in $(translations); do \
+    tx pull -l $$lang; \
+  done; \
+  $(MAKE) update-po
```
Comment (strk) [inline doc/Makefile.in]: (why) was this change needed ?

**Comment by strk** [comment]
Sorry, I've just realized the Japanese translation was not in the official repository at all !
It _is_ useful to add it to the repo, but if you modify it after pulling from transifex, you should also push the modified version. That's what I meant.

**Comment by boiledorange73** [comment]
OK !
I am relieved;-)

I have synchronized translations on transefix with ones on github.
There is no problem.

**Comment by strk** [comment]
Ok so I pulled the latest changes from transifex as of r14664 = 4f48d418b3450efade86ced84ec386638355bc89 (refs/remotes/trunk)

**Comment by boiledorange73** [comment]
Thank you !

It seems that I should not do pull request, but should make my effort to maintain translations *on transifex* and wait for merging.
Is this right ?

**Comment by strk** [comment]
PRs are also ok, in which case someone in PostGIS team will
need to merge them _and_ push them up transifex (with the "make
push-tx" call).

I understood you already pushed your changed up there, and for
that reason I pulled them directly. Was it a misunderstanding ?

**Comment by boiledorange73** [comment]
> PRs are also ok

I understand.

> I understood you already pushed your changed up there, and for that reason I pulled them directly. Was it a misunderstanding ?

NO, it was not a misunderstanding.

This request was rejected, so I thought request via github is not good for you.
I misunderstood ...
